apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.css

```diff
@@ -4,6 +4,10 @@
   font-size: inherit;
 }
 
+.pl-rich-text-editor-container .ql-tooltip {
+  z-index: 1000;
```

Comment:
Why so high? Quill uses `1` for other popover-type things, e.g. the font size pickers.

Reply:
Bootstrap uses values in the range of 1000 for dropdowns, modals and popovers, I was using that as a basis. But you're right, we don't need more than 1 here. Changed in 4c94e711d7574af13f0f7327197e70f7ff3025f4.